Add `add_english` function to install english locale

cycool29 opened the pull request:
Solve the `apt lock` not detecting error output correctly by changing terminal output locale temporarily.

Tested on vdesktop RPi OS (first use of vdesktop lol).

Botspot commented:
Are you saying that you found a situation in which the C locale did ***NOT*** produce a predictable output?

cycool29 commented:
> Are you saying that you found a situation in which the C locale did _**NOT**_ produce a predictable output?

Ya, I tested on a Spanish vdesktop and `C` locale seems not working.
After lots of googling I can't find what is actually `C` locale means and does.
So I think directly export `LANGUAGE` as `en_US` will be easier and more specific. 
Many error reports received with `apt_lock_wait` passed but still show message `Could not get lock...` and the problem probably because of the locale/language problem.

cycool29 commented:
@Botspot Any ideas?

Botspot commented:
I've fixed two issues:

1. If for some reason `/usr/share/i18n/SUPPORTED` returns multiple matches, it would most likely return a bash error. This is fixed by placing the variable in quotation marks when it's being assigned. Also I went ahead and removed the unnecessary variable `english_supported` - only used once.
2. Grep had regex (regular expression) enabled for no reason. The period character (`.`) is a regex symbol for "*match any character*". 
    Here's a possible failure scenario:
    ```
    echo 'en_USaUTF-8' | grep -o 'en_US.UTF-8' #match found
    ```
    To fix, add the `-F` flag.
    ```
    echo 'en_USaUTF-8' | grep -oF 'en_US.UTF-8' #match not found
    ```

I've gone ahead and added both fixes, and this PR will be merged.

cycool29 commented:
> I've fixed two issues:
> 
> 1. If for some reason `/usr/share/i18n/SUPPORTED` returns multiple matches, it would most likely return a bash error. This is fixed by placing the variable in quotation marks when it's being assigned. Also I went ahead and removed the unnecessary variable `english_supported` - only used once.
> 2. Grep had regex (regular expression) enabled for no reason. The period character (`.`) is a regex symbol for "_match any character_".
>    Here's a possible failure scenario:
>    ```
>    echo 'en_USaUTF-8' | grep -o 'en_US.UTF-8' #match found
>    ```
>    
>    
>        
>          
>        
>    
>          
>        
>    
>        
>      
>    To fix, add the `-F` flag.
>    ```
>    echo 'en_USaUTF-8' | grep -oF 'en_US.UTF-8' #match not found
>    ```
> 
> I've gone ahead and added both fixes, and this PR will be merged.

Thanks!

theofficialgman commented:
@cycool29 I am getting a very strange error from a user as a result of this change

.NET CLI sees the english locale being available and changes to using it automatically for some reason which is annoying.

I can't reproduce any issues with using the C locale and not generating english even when only using a system with the C locale and spanish (es_US) available.

can you please elaborate on your test setup as I can't replicate
